packages/mask/src/plugins/Tips/components/TipButton.tsx

```diff
@@ -4,22 +4,24 @@ import { useCurrentWeb3NetworkPluginID, useSocialAddressListAll } from '@masknet
 import { EMPTY_LIST, NextIDPlatform, ProfileIdentifier } from '@masknet/shared-base'
 import { makeStyles, ShadowRootTooltip } from '@masknet/theme'
 import { NextIDProof } from '@masknet/web3-providers'
-import { NetworkPluginID } from '@masknet/web3-shared-base'
+import { NetworkPluginID, SocialAddressType } from '@masknet/web3-shared-base'
 import type { TooltipProps } from '@mui/material'
 import classnames from 'classnames'
-import { uniq } from 'lodash-unified'
+import { uniqBy } from 'lodash-unified'
 import { FC, HTMLProps, MouseEventHandler, useCallback, useEffect, useMemo } from 'react'
 import { useAsyncRetry } from 'react-use'
 import { MaskMessages } from '../../../../shared'
 import { useCurrentVisitingIdentity } from '../../../components/DataSource/useActivatedUI'
-import Services from '../../../extension/service'
 import { activatedSocialNetworkUI } from '../../../social-network'
+import { useProfilePublicKey } from '../hooks/useProfilePublicKey'
 import { usePublicWallets } from '../hooks/usePublicWallets'
 import { useI18N } from '../locales'
 import { PluginNextIDMessages } from '../messages'
+import type { AddressConfig } from '../types'
 
 interface Props extends HTMLProps<HTMLDivElement> {
-    addresses?: string[]
+    addresses?: AddressConfig[]
+    recipient?: AddressConfig['address']
```

Comment:
```suggestion
    recipient?: string
```

Reply:
I prefer to reuse typing. When `address` in `AddressConfig` gets updated, no need to update this prop as well.